nerves_toolchain_aarch64_nerves_linux_musl/defconfig

```diff
@@ -0,0 +1,10 @@
+CT_LOCAL_TARBALLS_DIR="${CT_TOP_DIR}/../dl"
+CT_ARCH_ARM=y
+CT_ARCH_64=y
+CT_KERNEL_LINUX=y
+CT_LINUX_V_4_19=y
+CT_MULTILIB=y
+CT_EXPERIMENTAL=y
+CT_TARGET_VENDOR="multilib"
+CT_LIBC_MUSL=y
+CT_GCC_V_11=y
```

Comment:
```suggestion
CT_EXPERIMENTAL=y
CT_LOCAL_TARBALLS_DIR="${CT_TOP_DIR}/../dl"
CT_ARCH_ARM=y
CT_ARCH_64=y
CT_KERNEL_LINUX=y
CT_LINUX_V_4_19=y
CT_LIBC_MUSL=y
CT_GCC_V_11=y
```

Do you need multilib? Typically we don't build multilib toolchains for Nerves.

Also, if you put `CT_EXPERIMENTAL=y` at the top, it will get past the unmerge script since ctng likes the option first.